cmd/godmine/main.go

```diff
@@ -262,7 +270,10 @@ func createIssue() {
 }
 
 func updateIssue(id int) {
-	c := redmine.NewClient(conf.Endpoint, conf.Apikey)
+	c, err := redmine.NewClient(conf.Endpoint, auth)
```

Comment:
🚫 **[checkstyle]** <staticcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1019: redmine.NewClient is deprecated: Use redmine.ClientBuilder to create a redmine client that supports more options and detects configuration mistakes.